acceptance/bundle/validate/include_locations/output.dev.json

```diff
@@ -72,11 +72,6 @@
         2,
         6,
         11
-      ],
```

Comment:
This is interesting diff. Now, if "normalization" doesn't change a value, we discard its updates to locations. In this case, after merging effective value didn't change because both branches had the same value.